railib/__init__.py

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version_info__ = (0, 7, 0)
+__version_info__ = (0, 7, 3)
```

Comment:
@joshuafcole Regarding the version bump: I actually _neglected_ to bump; it shows as a double-bump because of a Git error on my part that apparently got handled by GitHub in kind of a weird way. (Namely, I forgot to go back to `main` before creating my new branch and instead used my previous branch.)

Anyway, it should be OK now, GitHub's interface weirdness notwithstanding.

Reply:
Hey, the diff shows correctly now that I merged main 🤷

Reply:
Sounds good, did you want to cut a release or should I?

Reply:
Shoot; good catch. I thought I did this but apparently not.

Done now.